tests/test_qr_to_waypoint_names.py

```diff
@@ -0,0 +1,60 @@
+"""
+Testcases for parsing QR to waypoint.
+"""
+
+
+from modules.qr_to_waypoint_names import qr_to_waypoint_names
```

Comment:
Prefer `from modules import qr_to_waypoint_names` and then use `qr_to_waypoint_names.` prefix for namespace consideration.